src/test/resources/google/cloud/compute/v1/compute.proto.baseline

```diff
@@ -1,3 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated by the disco-to-proto3-converter. DO NOT EDIT!
```

Comment:
Yeah, I think something up here like the following might callmroe attention to this message. WDYT?

```
// ** DO NOT EDIT! **
// This proto was generated from a Google API Discovery file by the disco-to-proto3-converter
// for use by GAPIC generators.
//   Source Discovery file: compute.v1.small.json
//   Source file revision: 20200302
//   API name: compute
//   API version: v1
```

Reply:
As  menitoned before, I was following the protoc example, trying to make it consistent with the other files. Unless you have strong opinion about it, let's keep it as is for now. We can always revise it later (simple change in any case).